airflow/models/taskinstance.py

```diff
@@ -1650,6 +1652,8 @@ def _execute_task(self, context, task_orig):
         except:  # noqa: E722
             _TASK_EXECUTION_FRAME_LOCAL_STORAGE.frame = currentframe()
             raise
+        finally:
+            del _TASK_EXECUTION_FRAME_LOCAL_STORAGE.in_user_code
```

Comment:
Should we set this to False instead of deleting it? Might not actually matter, but could let us simplify `get_truncated_error_traceback`?

Reply:
We could -- I'm not sure why I chose delete than set to False tbh.

Reply:
Done